diff --git a/node_modules/@cloudflare/pages-functions-compiler/bin/plinko.mjs b/node_modules/@cloudflare/pages-functions-compiler/bin/plinko.mjs
index e47e3b7..0151f99 100755
--- a/node_modules/@cloudflare/pages-functions-compiler/bin/plinko.mjs
+++ b/node_modules/@cloudflare/pages-functions-compiler/bin/plinko.mjs
@@ -146,6 +146,8 @@ export const routes = [
 import path2 from "path";
 import { build } from "esbuild";
 import { fileURLToPath } from "url";
+import nodeModulesPolyfill from "@esbuild-plugins/node-modules-polyfill";
+import nodeGlobalsPolyfill from "@esbuild-plugins/node-globals-polyfill";
 var __dirname = fileURLToPath(path2.dirname(import.meta.url));
 function buildWorker({
   routesModule,
@@ -155,6 +157,12 @@ function buildWorker({
   console.log(`Compiling worker to "${outfile}"`);
   return build({
     entryPoints: [path2.resolve(__dirname, "../templates/plinko.worker.ts")],
+    plugins: [
+      nodeModulesPolyfill.NodeModulesPolyfillPlugin(),
+      nodeGlobalsPolyfill.NodeGlobalsPolyfillPlugin({
+        buffer: true,
+      }),
+    ],
     inject: [routesModule],
     bundle: true,
     format: "esm",
